### PR TITLE
Update explicit disclosure docs for 2.8.0

### PR DESCRIPTION
**Pull request by tudor-da**

### CODE REVIEW

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 1]
```diff
@@ -234,15 +224,14 @@ the original `CreatedEvent` (see above):
 
 - **template_id** - The contract's template id.
 - **contract_id** - The contract id.
-- **arguments** - The contract's create arguments. This field is a protobuf ``oneof``
-  and it allows either passing the contract's create arguments typed (as ``create_arguments``)
-  or as a byte array (as ``create_arguments_blob``).
-  Generally, clients should use the ``create_arguments_blob`` for convenience since they can be received as such
-  from the stakeholder off-ledger (see above).
-- **metadata** - The contract metadata. This field can be populated as received from the stakeholder (see below).
+- **created_event_blob** - The contract's representation as an opaque blob encoding. This field is populated **only** on demand for ``GetTransactions`` and ``GetTransactionTrees`` streams (read more about :ref:`configuring transaction filters <transaction-filter>`).
```
Comment (mziolekda) [inline docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:227]: One part of this paragraph belongs in the previous chapter where you say that the contracts can be fetched in the LAPI queries

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 2]
```diff
@@ -267,16 +255,13 @@ with the Ledger API running on port ``5031``.
...
+  priceQuotationQuery=$(grpcurl -plaintext -d '{"ledgerId":"stockExchangeParticipant","begin":{"absolute":"0000000000000000"},"end":{"boundary":"LEDGER_END"},"filter":{"filters_by_party":{"'"$stockExchangeId"'":{"inclusive":{"template_filters":[{"template_id":{"package_id":"'"$packageId"'","module_name":"StockExchange","entity_name":"PriceQuotation"},"include_created_event_blob":true}]}}}},"verbose":true}' localhost:5011 com.daml.ledger.api.v1.TransactionService/GetTransactions)
```
Comment (mziolekda) [inline docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:261]: In order to make sense of this example, we should say something about the topology, which party is connected to which participant. I am a bit confused about the fact that we have 3 different participant addresses here 5011 5041 and 5031

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 3]
```diff
@@ -267,16 +255,13 @@ with the Ledger API running on port ``5031``.
...
+  stockQuery=$(grpcurl -plaintext -d '{"ledgerId":"stockExchangeParticipant","begin":{"absolute":"0000000000000000"},"end":{"boundary":"LEDGER_END"},"filter":{"filters_by_party":{"'"$stockExchangeId"'":{"inclusive":{"template_filters":[{"template_id":{"package_id":"'"$packageId"'","module_name":"StockExchange","entity_name":"Stock"},"include_created_event_blob":true}]}}}},"verbose":true}' localhost:5011 com.daml.ledger.api.v1.TransactionService/GetTransactions)
```
Comment (mziolekda) [inline docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:258]: As discussed off-line, I would use pretty-printed json

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst (hunk 2 above)
Comment (tudor-da) [reply docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst]: Added 👍

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst (hunk 1 above)
Comment (tudor-da) [reply docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst]: Adapted 👍

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst (hunk 3 above)
Comment (tudor-da) [reply docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst]: Pretty-printed 👍

**Review by mziolekda — APPROVED** [review]

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 4]
```diff
@@ -234,49 +235,180 @@ the original `CreatedEvent` (see above):
...
-- **arguments** - The contract's create arguments. This field is a protobuf ``oneof``
-  and it allows either passing the contract's create arguments typed (as ``create_arguments``)
-  or as a byte array (as ``create_arguments_blob``).
-  Generally, clients should use the ``create_arguments_blob`` for convenience since they can be received as such
-  from the stakeholder off-ledger (see above).
-- **metadata** - The contract metadata. This field can be populated as received from the stakeholder (see below).
+- **created_event_blob** - The contract's representation as an opaque blob encoding.
+
+.. note:: Only contracts created starting with Canton 2.8 can be shared as disclosed contracts.
+  Prior to this version, contracts' **CreatedEvent** does not have the required `created_event_blob` field populated
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:241]: (it doesn't seem necessary to me to specify that the CreatedEvent is about contracts -- I think that's clear from the preceding sentence. But let me know if this revision doesn't work for you)

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst [hunk 5]
```diff
@@ -3,29 +3,23 @@
 
 .. _explicit-contract-disclosure:
 
-Explicit Contract Disclosure (Alpha)
+Explicit Contract Disclosure (Beta)
 ###########################################
 
-In Daml, you must specify upfront who can view data using `observer` annotations on contracts. To change who can see the data, you would typically need to rewrite the contract (eg an asset) with a new annotation. Canton 2.7 introduces explicit contract disclosure as a feature that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution. This supports efficient, scalable data sharing on the ledger. 
+In Daml, you must specify upfront who can view data using `observer` annotations on contracts.
+To change who can see the data, you would typically need to rewrite the contract (e.g. an asset) with a new annotation.
```
Comment (marciw-da) [inline docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:10]:
```suggestion
To change who can see the data, you would typically need to rewrite the contract (asset) with a new annotation.
```

Comment (marciw-da) [reply docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:10]: I think this is what you mean, but let me know if I've misunderstood

**Review by marciw-da — COMMENTED** [review]
Thanks for your patience! Made some suggestions.

File: docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (tudor-da) [reply docs/2.8.0/docs/app-dev/explicit-contract-disclosure.rst:241]: good observation, mentioning `contract` was superfluous

**Review by marciw-da — APPROVED** [review]
LGTM. Thanks!

**Comment by tudor-da** [comment]
@marciw-da So **build_docs** fails now because I left the Explicit Disclosure page out of the ToC. Are you aware of a workaround for this? If not, I can read around

**Comment by tudor-da** [comment]
@marciw-da @mziolekda I have moved the page to 3.0.0 as discussed. Please re-review

**Pull request merged**